lib/inc/internal/facts/linux/virtualization_resolver.hpp

```diff
@@ -7,6 +7,8 @@
 #include "../resolvers/virtualization_resolver.hpp"
 #include <string>
 
+using namespace std;
```

Comment:
Please don't do this in header files. It can cause surprises for anyone trying to use the header file.